src/sentry/web/client_config.py

```diff
@@ -430,6 +430,9 @@ def get_context(self) -> Mapping[str, Any]:
             "isOnPremise": is_self_hosted(),
             "isSelfHosted": is_self_hosted(),
             "isSelfHostedErrorsOnly": is_self_hosted_errors_only(),
+            # sentryMode intends to supersede isSelfHosted,
+            # so we can differentiate between "SELF_HOSTED", "SINGLE_TENANT", and "SAAS".
+            "sentryMode": settings.SENTRY_MODE.value.upper(),
```

Comment:
does upper do anything here?  could just use `name` ?

Reply:
StrEnum's auto makes everything lowercase 😆

Reply:
but `name` should be upper no?

Reply:
ah you're right